docs/docsite/rst/community/index.rst

```diff
@@ -8,8 +8,72 @@ Welcome to the Ansible Community Guide!
 
 The purpose of this guide is to teach you everything you need to know about being a contributing member of the Ansible community.
 
+All types of contributions are welcome, and necessary to Ansible's continued success.
+
 To get started, please read and understand the :ref:`code_of_conduct`, and then select one of the following topics.
 
+
+Getting Started
+===============
+
```

Comment:
easy ways you can contribute?https://docs.ansible.com/ansible/devel/community/how_can_I_help.html